### PR TITLE
docs: add the guide for contributors

### LINKED ISSUE #5: Add contributors guide
We need to implement a contributors guide for new contributors.
Feel free to propose.

### PR DESCRIPTION
**Pull request by salazarsebas**
# Pull Request Overview

## 📝 Summary
Create a guide for developers who want to contribute to the project. Generating a general nomenclature and rules.

### 🔗 Related Issues
- Closes #5  .

## 🔄 Changes Made
Add contribution guide.

## 🖼️ Current Output
Contributors Guide PayStell

## 💬 Comments
I hope the solution solves what you requested, please let me know of any changes I need to make. Best regards.

### CODE REVIEW

**Comment by salazarsebas** [comment]
Please tell me what you think of the pr. @MPSxDev  ⛩️🚀⌨️

File: CONTRIBUTORS_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- Make changes in your local repository
+- Follow **atomic commit principles**:
+
+   - Each commit should address a single, logical change.
+   - Avoid bundling unrelated changes in a single commit. In case you want to add additional items, please ask the mainteiners first.
+   - Write clear and descriptive commit messages using the format:
+
+   ```bash
+   git add .
+   git commit -m "[type] description"
```
Comment (MPSxDev) [inline CONTRIBUTORS_GUIDE.md:61]: Remove [ ]. Use prefixes like feat: or fix: for all types.

File: CONTRIBUTORS_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Feel welcome and read the following sections in order to know how to ask questions and how to work on something.
+
+Please make sure you are welcoming and friendly in all of our spaces.
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
+
+## 👨🏻‍💻 Issues
+
+The best way to contribute to our projects is to apply to the issues <a href="https://github.com/PayStell/paystell-website/issues" target="_blank">here</a> .
+
```
Comment (MPSxDev) [inline CONTRIBUTORS_GUIDE.md:16]: add: PRs are not allowed; all PRs must originate from an assigned issue.

File: CONTRIBUTORS_GUIDE.md (hunk 1 above)
Comment (salazarsebas) [reply CONTRIBUTORS_GUIDE.md:61]: Okay, I'm going to work on that.

File: CONTRIBUTORS_GUIDE.md (hunk 2 above)
Comment (salazarsebas) [reply CONTRIBUTORS_GUIDE.md:16]: Okay, I'm going to work on that.

**Review by MPSxDev — APPROVED** [review]
Perfect!

**Pull request merged**